assets/styles/main.css

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
If you're using a preprocessor, you technically don't need to track your output css files (just the scss files), but it's up to you